### ISSUE TITLE
Error migrating issues to GitHub

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Austin503/support-tools#33]
```
What steps will reproduce the problem?
1. Try to export to GitHub
2. Wait for failure email

What is the expected output? What do you see instead?
Not sure, I haven't managed to make the exporter run correctly.

What version of the product are you using? On what operating system?
Yours? OSX? 

Please provide any additional information below.
Tried to use the standard export option, failed to migrate issues. The repo is 
mlode. 

```

Original issue reported on code.google.com by `richard....@gmail.com` on 14 Mar 2015 at 6:02

**Comment by GoogleCodeExporter** [comment]
```
Looking at our logs, it appears to be a bug in the Google Code importer. We'll 
take a look and update this bug when it is fixed. When we have resolved this 
bug, you can delete the half-imported GitHub repo and retry the Google Code 
Exporter.

From the logs:
creating GitHub issue: POST 
https://api.github.com/repos/RichardLitt/mlode/issues: 422 Validation Failed 
[{Resource:Label Field:name Code:invalid}]

```

Original comment by `chrsm...@google.com` on 14 Mar 2015 at 6:48
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I had the same problem with the repo "droidwall". Can you please check if it is 
the same root cause or if I should open a new issue?
Thanks
```

Original comment by `rodrigo...@gmail.com` on 23 Mar 2015 at 4:59

**Comment by GoogleCodeExporter** [comment]
```
Still don't have an answer on this, guys. 
```

Original comment by `richard....@gmail.com` on 24 Mar 2015 at 3:12

**Comment by GoogleCodeExporter** [comment]
```
It looks like it might be a bug with the duplicate label Dataset-WordNet on 
https://code.google.com/p/mlode/issues/detail?id=98

Could you remove one instance of that label and retry the export? We can 
probably handle this case better on our side too, but removing the duplicate 
label might unblock you quicker.
```

Original comment by `jasonhall@google.com` on 24 Mar 2015 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
I can't figure out how to remove a label. Sorry about that.
```

Original comment by `richard....@gmail.com` on 25 Mar 2015 at 2:39

**Comment by GoogleCodeExporter** [comment]
```
The exporter should now handle duplicate labels better, can you try again even 
without removing the label?
```

Original comment by `jasonhall@google.com` on 25 Mar 2015 at 2:53

**Comment by GoogleCodeExporter** [comment]
```
FYI, it worked now for my project (droidwall). Hope it works for you too.
Thanks!
```

Original comment by `rodrigo...@gmail.com` on 25 Mar 2015 at 2:58

**Comment by GoogleCodeExporter** [comment]
```
It works! Thanks!
```

Original comment by `richard....@gmail.com` on 25 Mar 2015 at 2:00

**Comment by GoogleCodeExporter** [comment]
Original comment by `chrsm...@google.com` on 14 Apr 2015 at 4:23
- Changed state: **Fixed**
